Assets/Scripts/Player/PlayerProperties.cs

```diff
@@ -16,4 +17,15 @@ public static class PlayerProperties
     public static float colorChangeCooldown = 10f;
     public static float colorChangeCountdown = 10f;
     public static int distance = 0;
+
+    private static Vector3 checkpoint;
```

Comment:
Prečo set get a nie len public ako všetko ostatné?

Reply:
Pretoze sa to bude aj tak prerabat.

Reply:
Rozprávali sme sa dávnejšie s Jakubom, že by to bolo z programátorského hľadiska rozumnejšie, logickejšie a lepšie dodržiavať princípy enkapsulácie. Preto settery a gettery a nie všetko public static. Keď je všetko public static, tak sa to na pozadí nekontrolovateľne prepisuje a niekedy sa to správa divne.

Reply:
Ok, môže byť. Potom ale v C# sa nepíšu gettre a settre ale jednoducho cez autoproperty `public static Vector3 Checkpoint { get; set; }`

Reply:
A tým pádom by sa mal neskôr doplniť get set podľa potreby na všetky ostatné fieldy.